tls/s2n_config.h

```diff
@@ -50,6 +50,11 @@ struct s2n_config {
     s2n_cert_auth_type client_cert_auth_type;
     verify_cert_trust_chain_fn *verify_cert_chain_cb;
     void *verify_cert_context;
+
+    uint8_t mfl_code;
+
+    /* if this is FALSE, server will ignore client's Max Fragment Length request */
+    int enable_server_mfl;
```

Comment:
You might be able to change this to "enable_mfl" and use it for both client and server side and eliminate the need for S2N_TLS_MAX_FRAG_LEN_EXT_NONE